lib/model/folder_sendrecv.go

```diff
@@ -989,7 +989,7 @@ func (f *sendReceiveFolder) renameFile(cur, source, target protocol.FileInfo, sn
 		err = errModified
 	default:
 		var fi protocol.FileInfo
-		if fi, err = scanner.CreateFileInfo(stat, target.Name, f.mtimefs); err == nil {
```

Comment:
In these places we use the `SyncOwnership` bool instead of `ScanOwnership`, because it's about detecting whether the file is different from what we've synced previously. If we aren't syncing ownership we don't need to load ownership info because we're not going to compare it.